.github/workflows/tests.yaml

```
@@ -137,6 +142,10 @@ jobs:
     - name: ${{ matrix.test.name }}
       run: ${{ matrix.test.command }}
       env: ${{ matrix.test.env }}
+    - name: Print stderr logs
+      run: grep -H "" *.stderr.log || true
```

Is this like for debugging purposes? If so let's make sure it is run everytime.
```suggestion
      run: grep -H "" *.stderr.log || true
      if: always()
```

---

It's for debugging the warnings (from the step below). If there is an error before there is no need to print those stderr, they already appear in regular CI output